src/rpc/server.cpp

```diff
@@ -484,10 +501,9 @@ UniValue CRPCTable::execute(const JSONRPCRequest &request) const
     if (!pcmd)
         throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Method not found");
 
-    g_rpcSignals.PreCommand(*pcmd);
-
     try
     {
+        auto active_command = g_active_commands_counter;
```

Comment:
`active_command` unused? :-)

Reply:
Ya, the idea is to have a RAII counter and for now I'm using the shared ptr usage counter 😅